PayoneerCheckout.podspec

```
@@ -8,14 +8,8 @@ Pod::Spec.new do |s|
   s.author           = { 'Payoneer Germany GmbH' => '' }
   s.source           = { :git => 'https://github.com/optile/checkout-ios.git', :tag => s.version.to_s }
 
-  s.ios.deployment_target = '12.4'
-  s.swift_version = '5.3'
+  s.ios.deployment_target = '12.0'
+  s.swift_version = '5.5'
 
   s.source_files = 'Sources/**/*.swift'
-  
-  s.resources = ['Sources/Assets/*']
```

I think we need to specify the new path to resources here, why did you delete it?

```suggestion
  s.resources = ['Sources/Resources/*']
```

---

Good catch!

---

I believe this is only needed if you want to expose resources to the host app. Everything works after removing it, but I'll  test some more.

---

The framework automatically downloads images when they're not found locally. Please check, if the framework is not downloading it from the internet after disabling this, it is okay.

---

You could add a breakpoint here to check this:
https://github.com/optile/checkout-ios/blob/4568c6205bea169959be3f59fbb4011e281db7fe/Sources/UI/Service/PaymentSessionService/DataDownloadProvider.swift#L30-L37

And check output as well it should be triggered as well: https://github.com/optile/checkout-ios/blob/4568c6205bea169959be3f59fbb4011e281db7fe/Sources/Network/Requests/DownloadData.swift#L31

---

Updated in https://github.com/optile/checkout-ios/pull/158/commits/7e4d918829d1992e9b66075d4c4b2202e58e82a3. Thanks for pointing this out and for the thorough review!